module/applications/actor/movement-config.mjs

```diff
@@ -5,34 +5,37 @@ import BaseConfigSheet from "./base-config.mjs";
  */
 export default class ActorMovementConfig extends BaseConfigSheet {
 
-  /** @override */
+  /** @inheritdoc */
   static get defaultOptions() {
     return foundry.utils.mergeObject(super.defaultOptions, {
       classes: ["dnd5e"],
       template: "systems/dnd5e/templates/apps/movement-config.hbs",
       width: 300,
-      height: "auto"
+      height: "auto",
+      sheetConfig: false
     });
   }
 
   /* -------------------------------------------- */
 
-  /** @override */
+  /** @inheritdoc */
   get title() {
     return `${game.i18n.localize("DND5E.MovementConfig")}: ${this.document.name}`;
   }
 
   /* -------------------------------------------- */
 
-  /** @override */
+  /** @inheritdoc */
   getData(options={}) {
     const source = this.document.toObject();
+    const isActor = this.document instanceof Actor;
+    const keyPath = isActor ? "system.attributes.movement" : "system.movement";
```

Comment:
See similar comment for `ActorTypeConfig` about the merits of passing in `keyPath` as an option to the constructor.